packages/trie/test/index.spec.ts

```diff
@@ -17,336 +17,365 @@ import { bytesToNibbles } from '../src/util/nibbles.js'
 
 import type { HashKeysFunction } from '../src/index.js'
 
-for (const cacheSize of [0, 100]) {
-  describe('simple save and retrieve', () => {
-    it('should not crash if given a non-existent root', async () => {
-      const root = hexToBytes('0x3f4399b08efe68945c1cf90ffe85bbe3ce978959da753f9e649f034015b8817d')
-
-      const trie = new Trie({ root })
-      const value = await trie.get(utf8ToBytes('test'))
-      assert.equal(value, null)
-    })
-
-    const trie = new Trie({ cacheSize })
-
-    it('save a value', async () => {
-      await trie.put(utf8ToBytes('test'), utf8ToBytes('one'))
-    })
-
-    it('should get a value', async () => {
-      const value = await trie.get(utf8ToBytes('test'))
-      assert.equal(bytesToUtf8(value!), 'one')
-    })
+for (const keyPrefix of [undefined, hexToBytes('0x1234')]) {
```

Comment:
looks like here as well this is the only diff and rest is formatting change :+1: